Formula/berkeley-db5.rb

```diff
@@ -0,0 +1,67 @@
+class BerkeleyDb5 < Formula
```

Comment:
`BerkeleyDbAT5` and rename the formula to `berkeley-db@5.rb`. For bonus points: make the same change to `berkeley-db4.rb` and add it to `formula_renames.json`